### ISSUE TITLE
Fix game mode setting (also neaten script importing)

### ISSUE THREAD

**Issue by swalladge** [piqueserver/piqueserver#165]
ctf and tc game modes are built in, other modes require a script with that name to control it. With how recent changes to script handling has made it not possible to import a script directly by there name. It requires setting script names like `piqueserver.scripts.votekick` instead of just `votekick`, which is great because now we can install scripts like any other python package and use them.

However this means that setting a custom game mode also requires `piqueserver.scripts.tdm` instead of just `tdm`...

This needs to either be thoroughly documented, or work done to make it neater to include scripts (perhaps with some importlib magic).

**Comment by NotAFile** [comment]
The idea was this would be solved by https://github.com/piqueserver/piqueserver/issues/109

Perhaps as a preliminary hack, it would be nice to at least display a nice value on the server list for this. Or do you generally mean being able to use it as "ctf"?

I don't think that's a good idea or possible, because how would that work for several packages? If there's another package/repo that provides an updated/improved CTF mode, how does it tell the difference?

Some import magic could be done though: If the script is not found, the piqueserver namespace could be prepended and retried.

**Comment by swalladge** [comment]
Yeah either way, as long as it's documented. It would be nice to have pretty values for the statusserver and other places they are referenced though. Perhaps game modes and scripts could have optional global variables or attributes for the display names and info.

**Comment by NotAFile** [comment]
A global attribute on the script module, e.g. GAMEMODE_NAME would be a simple option.

I think also another thing that might be worth doing is to add module-level constants `PIQUE_SCRIPT_MODULE = "scripts", PIQUE_MAP_LOCATION = "maps"` and then use those to resolve the true location of the scripts/gamemodes/maps, that way it's only `piqueserver.ctf` instead of `piqueserver.gamemodes.ctf`.

Also look at what ace.py has done: https://github.com/10se1ucgo/ace.py/blob/master/acemodes/ctf.py

That's where I wanted to go with #109.
